schemas/content-fragments/hotels/hotel.ts

```diff
@@ -1,62 +1,196 @@
+import { holidayExperiences, holidayThemes, hotelTypes, propertyCategories } from "../../constants";
+
 export default {
   name: "hotel",
   title: "Hotels",
   type: "document",
   fields: [
     {
-      name: "propertyId",
-      title: "Property Id",
+      name: "hotelName",
+      title: "Name",
       type: "string",
     },
     {
-      name: "propertyType",
-      title: "Property Type",
+      name: "hotelId",
+      title: "Id",
       type: "string",
     },
     {
-      name: "propertySubType",
-      title: "Property Sub Type",
+      name: "brandId",
+      title: "Brand Id",
       type: "string",
     },
     {
-      name: "propertyDetails",
-      title: "Property Details",
-      type: "hotelDetails",
+      name: "hotelDescription",
+      title: "Description",
+      type: "string",
     },
     {
-      name: "propertyAddress",
-      title: "Property Address",
+      name: "hotelPath",
+      title: "URL",
       type: "string",
     },
     {
-      name: "propertyContact",
-      title: "Property Contact",
-      type: "string",
+      name: "taxonomies",
+      title: "Taxonomies",
+      type: "object",
+      fields: [
+        {
+          name: "propertyCategory",
+          title: "Category",
+          type: "string",
+          options: { list: propertyCategories }
+        },
+        {
+          name: "hotelType",
+          title: "Hotel Type",
+          type: "string",
+          options: {
+            list: hotelTypes,
+          },
+        },
+        {
+          name: "holidayExperience",
+          title: "Holiday Experience",
+          type: "string",
+          options: {
+            list: holidayExperiences,
+          },
+        },
+        {
+          name: "holidayTheme",
+          title: "Holiday THeme",
+          type: "string",
+          options: {
+            list: holidayThemes,
+          },
+        },
+      ]
     },
     {
-      name: "propertyAvailability",
-      title: "Property Availability",
+      name: "hotelSubType",
+      title: "Sub Type",
       type: "string",
     },
     {
-      name: "propertyFacilities",
-      title: "Property Facilities",
-      type: "string",
+      name: "hotelOverview",
```

Comment:
Same here as well. type and name have common nomenclature. Please change it.

Reply:
resolved , please check